app/celery/research_mode_tasks.py

```diff
@@ -22,21 +28,8 @@
 
 
 def send_sms_response(provider, reference, to):
-    headers = {"Content-type": "application/x-www-form-urlencoded"}
-    body = firetext_callback(reference, to)
-    # to simulate getting a temporary_failure from firetext
-    # we need to send a pending status updated then a permanent-failure
-    if body["status"] == "2":  # pending status
-        make_request(SMS_TYPE, provider, body, headers)
-        # 1 is a declined status for firetext, will result in a temp-failure
-        body = {
-            "mobile": to,
-            "status": "1",
-            "time": "2016-03-10 14:17:00",
-            "reference": reference,
-        }
-
-    make_request(SMS_TYPE, provider, body, headers)
+    body = aws_ses_callback(reference, to)
```

Comment:
It should be an SNS callback for SMS

Reply:
Hmm funny I named it this way, the code is aligned with SNS callbacks.